RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -31,6 +31,10 @@ public class PaywallViewController: UIViewController {
         return .fullScreen
     }
 
+    var shouldAddTopConstraint: Bool {
```

Comment:
This seems unused?

Reply:
lol my bad, I added it and then realized that I already had the info in the mode variable